docs/UserGuide.md

```diff
@@ -94,22 +94,20 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing an internship entry : `edit`
 
-Edits an existing person in the address book.
+Edits an internship entry from the list of existing entries.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [r/ROLE] [s/STATUS ] [d/DATE]`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* The internship entry whose entry number is `INDEX` would be updated. `INDEX` needs to be a valid entry number as specified in the internship list displayed using `list` command. 
+* Items in square brackets are optional, but at least one of the optional fields must be provided.
+* `STATUS` must have one of the following values: `new`, `applied`, `assessment`, `interview`, `offered` or `rejected`.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 2 s/assessment r/SoftWare Developer` Sets the status and role of the internship entry 2 as `assessment` and `Software Developer` respectively.
```

Comment:
It should be `Software Developer` instead of `SoftWare Developer`, and would it better if we instead phrase it as "the second internship entry" instead of "internship entry 2"?